packages/dcos-integration-test/extra/conftest.py

```diff
@@ -195,6 +195,9 @@ def _wait_for_dcos(self):
         self._wait_for_srouter_slaves_endpoints()
         self._wait_for_metronome()
 
+    @retrying.retry(wait_fixed=2000,
```

Comment:
I 'd say 5 minutes should be plenty. We only do this after mesos has a master and adminrouter is up, at which point things should be moving along.